Library/Homebrew/build.rb

```diff
@@ -83,7 +83,8 @@ def install
       ENV.keg_only_deps = keg_only_deps
       ENV.deps = formula_deps
       ENV.run_time_deps = run_time_deps
-      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) }
+      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) } ||
+                deps.any? { |dep| dep.name == "linuxbrew/xorg/xorg" }
```

Comment:
No, because it gets pruned out by `Build.expand_reqs` https://github.com/Linuxbrew/brew/blob/master/Library/Homebrew/build.rb#L53